### DISCUSSION TITLE
[Mobile app]: Close modal when the back gesture/arrow is used

### DISCUSSION THREAD

**Discussion by matteosacchetto** [dailydotdev/daily#546] [Feature Requests]
One thing I think would improve mobile user experience, is the ability to close the new modal using the "back" gesture/button (I don't know its name, but I think you got the point)

I'm on Android and this is something which was possible with the previous version of article preview/discussion on a different page.

With the new version, on mobile I find myself accidentally closing the app instead of closing the modal since I try to close it using the back gesture/arrow, and It closes the app instead

If this is something you are already aware or duplicated, please feel free to close this discussion!

Keep up the good work 🚀

**Comment by idoshamun** [comment]
Hi @matteosacchetto,

Thank you for the suggestion. This can improve the mobile user experience significantly.
I'll report it back to the team. 😬

**Reply by omBratteng** [reply]
This would be highly appreciated. I suggested this to @rebelchris last night, as I encountered this issue when browsing app.daily.dev on my mobile.

I _think_ it would be a minor change, as when opening the modal, it changes the URL, but not pushing it as a change in the history.
Same thing also occurs when browsing on desktop, using the browsers back button, takes me to the previous page I was browsing before app.daily.dev.

**Reply by matteosacchetto** [reply]
As of today I've seen that it is now possible!!
Thank you, thank you, thank you!! 😍 🎉 🚀

**Reply by idoshamun** [reply]
Yap! I totally forgot to update you all :)
